packages/audio_room_with_chat/macos/Podfile

```
@@ -0,0 +1,42 @@
+platform :osx, '10.15'
```

```suggestion
platform :osx, '11.0'
```

Needs a higher minimum osx version.

I'm also fine with only keeping android and iOS folders, but if we add macos it should work.
We should really delete linux and windows as those are probably broken.